app/views/accounts/_pii.html.slim

```diff
@@ -40,21 +44,16 @@
       .sm-col.sm-col-4.px1.bold = t('account.index.phone')
       .sm-col.sm-col-8.px1 = pii.phone
 
-  .bg-gray-lighter.px2.py1.clearfix.fs-12p
-    - if !pii.obfuscated
-      .col.col-12.sm-col-6
-        = image_tag asset_url('lock.svg'), width: 8, class: 'mr1'
-        = t('account.security.text')
-      = link_to t('account.security.link'), MarketingSite.help_url
-
-    - elsif locked_for_session
-      .col.col-12.sm-col-6
-        = t('account.re_verify.footer')
-      .col.mx6
-        .usa-button.usa-button--outline.usa-button--small.col-md.grid-offset-4
-          = link_to t('account.verify'), '/reauthn'
-    - else
-      .col.col-12.sm-col-6
-        = image_tag asset_url('lock.svg'), width: 8, class: 'mr1'
-        = t('account.security.text')
+  - if !locked_for_session
```

Comment:
As a personal style thing, I try to have positive test cases first

```ruby
if locked_for_session
  # ...
else
  # ...
```

Reply:
There's no positive case here - but I can switch it to an `unless` if that's more clear

Reply:
Nah I must have gotten the `else` confused with something else. Personally I think `unless` makes things worse so I wouldn't sweat it